R/get_coverage_wcs.R

```diff
@@ -46,35 +47,56 @@
 #' }
 #'
 get_coverage_wcs <- function(
-    wcs = c("dtm", "dsm", "omz", "omw"),
+    wcs = c("dtm", "dsm", "omz", "omw", "dhmv"),
     bbox,
     layername,
     resolution,
-    wcs_crs = "EPSG:4258",
+    wcs_crs = c("EPSG:4258", "EPSG:31370"),
     output_crs = "EPSG:31370",
     bbox_crs = "EPSG:31370",
     version = c("1.0.0", "2.0.1"),
     ...) {
+
   # prelim check
   version <- match.arg(version)
+  wcs <- tolower(wcs) # case insensitive wcs
   wcs <- match.arg(wcs)
   wcs_crs <- match.arg(wcs_crs)
   bbox_crs <- match.arg(bbox_crs)
 
+
+  # constrain version | wcs
+  if (wcs == "dhmv") {
+    assert_that(version == "1.0.0",
+      msg = "WCS `DHMV` is incompatible with versions other than `1.0.0`.")
+  }
+
```

Comment:
I will test, thank you! Nevertheless, I feel it is useful to restrict versions to the ones we know that work? (better with a warning instead of an error)

Reply:
(I have converted this into a warning.)